util/misc/pre-commit-msg-hook

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# This is a git commit hook which automatically add a DCO signed-off
+# message if one is missing.  This message appends after the generation
+# of git default message.
+#
+# To utilize this script, place this file under .git/hooks/commit-msg
```

Comment:
Hi @soohoonchoi — I hadn't noticed until now that this script needs to change its name as well as its location when being installed.  Were there reasons that you didn't simply name the script `commit-msg`?  (not that I object, just curious).